docs/frameworks/symfony.md

```diff
@@ -154,46 +103,51 @@ To learn more about all this, read the [environment variables documentation](/do
 
 ## Trust API Gateway
 
-When hosting your site on Lambda, API Gateway will act as a proxy between the client and your function. 
+When hosting your site on Lambda, API Gateway will act as a proxy between the client and your function.
 
-By default, Symfony doesn't trust proxies for security reasons, but it's safe to do it when using API Gateway and
-Lambda. 
+By default, Symfony doesn't trust proxies for security reasons, but it's safe to do it when using API Gateway and Lambda.
 
 This is needed because otherwise, Symfony will not be able to generate URLs properly.
 
 You should add the following lines to `config/packages/framework.yaml`
 
 ```yaml
+# config/packages/framework.yaml
+
 framework:
-    # trust the remote address because API Gateway has no fixed IP or CIDR range that we can target
-    trusted_proxies: '127.0.0.1,REMOTE_ADDR'
-    # trust "X-Forwarded-*" headers coming from API Gateway
-    trusted_headers: ['x-forwarded-for', 'x-forwarded-proto', 'x-forwarded-port']
+  # trust the remote address because API Gateway has no fixed IP or CIDR range that we can target
+  trusted_proxies: '127.0.0.1,REMOTE_ADDR'
+  # trust "X-Forwarded-*" headers coming from API Gateway
+  trusted_headers: [ 'x-forwarded-for', 'x-forwarded-proto', 'x-forwarded-port' ]
 ```
 
-Note that API Gateway doesn't set the `X-Forwarded-Host` header, so we don't trust it by default. 
-You should only whitelist this header if you set it manually, for example in your CloudFront configuration (see how to 
-do it in the [example Cloudformation template](../websites.md#serving-php-and-static-files-via-cloudfront)).
+Note that API Gateway doesn't set the `X-Forwarded-Host` header, so we don't trust it by default. You should only whitelist this header if you set it manually,
+for example in your CloudFront configuration (see how to do it in
+the [example Cloudformation template](../websites.md#serving-php-and-static-files-via-cloudfront)).
 
 > Be careful with these settings if your app will not be executed only in a Lambda environment.
- 
+
 You can get more details in the [Symfony documentation](https://symfony.com/doc/current/deployment/proxies.html).
 
 ### Getting the user IP
 
-**When using CloudFront** on top of API Gateway, you will not be able to retrieve the client IP address and you will
-instead get one of Cloudfront's IP when calling `Request::getClientIp()`. If you really need this, you will need
-to whitelist [every CloudFront IP](https://docs.aws.amazon.com/AmazonCloudFront/latest/DeveloperGuide/LocationsOfEdgeServers.html)
+**When using CloudFront** on top of API Gateway, you will not be able to retrieve the client IP address, and you will instead get one of Cloudfront's IP when
+calling `Request::getClientIp()`. If you really need this, you will need to
+whitelist [every CloudFront IP](https://docs.aws.amazon.com/AmazonCloudFront/latest/DeveloperGuide/LocationsOfEdgeServers.html)
```

Comment:
I know you did not change this. But this is not really true. 

You can just update your index.php file with: 

```php
// Get user IP:
if (isset($_SERVER['LAMBDA_REQUEST_CONTEXT'])) {
    $context = json_decode($_SERVER['LAMBDA_REQUEST_CONTEXT'], true);
    $_SERVER['HTTP_X_FORWARDED_FOR'] = $context['identity']['sourceIp'] ?? '';
}
```

Reply:
There was some discussions about this on Slack and it seems to work fine but we weren't able to say if this is working in all scenarios. And I wasn't able to find AWS docs that states clearly what they put in it.

If you are sure about that, I'm a :100: to document it :+1: